### PR TITLE
Milestone three

### PR DESCRIPTION
**Pull request by SimonGrchevski**
- Create a table named owners 
- Create a table named species
- Modify animals table
- Populate owners and species tables
- Write the queries

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -11,3 +11,39 @@ INSERT INTO animals (name, date_of_birth, weight_kg,neutered, escape_attempts) V
...
 ('Blossom','1998-10-13',17.0,true,3);
+
+INSERT INTO owners(full_name, age) 
+VALUES 
+('Sam Smith',34),
+('Jennifer Orwell',19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
```
Comment (Emmyn5600) [inline data.sql]: - [x] Let us modify our insertion in this way

 INSERT INTO owners (full_name, age)
 VALUES ('Sam Smith', 34);

 INSERT INTO owners (full_name, age)
 VALUES ('Jennifer Orwell', 19);

 INSERT INTO owners (full_name, age)
 VALUES ('Bob', 45);

 INSERT INTO owners (full_name, age)
 VALUES ('Melody Pond', 77);

 INSERT INTO owners (full_name, age)
 VALUES ('Dean Winchester', 14);

 INSERT INTO owners (full_name, age)
 VALUES ('Jodie Whittaker', 38);

File: data.sql [hunk 2]
```diff
@@ -11,3 +11,39 @@ INSERT INTO animals (name, date_of_birth, weight_kg,neutered, escape_attempts) V
...
+
+INSERT INTO species(name)
+VALUES ('Pokemon'),('Digimon');
```
Comment (Emmyn5600) [inline data.sql]: - [x] Again, let us insert our data in this way

 INSERT INTO species (name)
 VALUES ('Pokemon');

 INSERT INTO species (name)
 VALUES ('Digimon');

File: data.sql [hunk 3]
```diff
@@ -11,3 +11,39 @@ INSERT INTO animals (name, date_of_birth, weight_kg,neutered, escape_attempts) V
...
+SET owner_id = 5
+WHERE name = 'Angemon' OR name = 'Boarmon';
+
+UPDATE animals
+SET species_id = 2
+WHERE name like '%mon';
+
+UPDATE animals
+SET species_id = 1 
+WHERE species_id IS NULL;
```
Comment (Emmyn5600) [inline data.sql]: - [x] Let us modify our implementation here, you  are updating animals after  setting the owner_id to the value, imagine once you have many ids how would you handle it?

Hint: refer to following sample solution to answer the remaining:

`UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';`

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @SimonGrchevski  

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Changes required :writing_hand: 

         - Please work on the review too

    Cheers and Happy coding!:clap::clap::clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: data.sql (hunk 1 above)
Comment (sinansevgi) [reply data.sql]: :heavy_check_mark: This is more readable now :+1:

File: data.sql (hunk 2 above)
Comment (sinansevgi) [reply data.sql]: :heavy_check_mark: Well resolved

File: data.sql (hunk 3 above)
Comment (sinansevgi) [reply data.sql]: :+1:

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @SimonGrchevski , 

Great work on making changes requested by the previous code reviewer.
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**